### ISSUE TITLE
Isometric Dice

### ISSUE THREAD

**Issue by ChristopherTheBoss** [Templarian/MaterialDesign#6476]
### Before submitting a new icon request, please confirm the following:

- [X] I have [searched all issues](https://github.com/Templarian/MaterialDesign/issues?q=is%3Aissue) to make sure there isn't a request for this icon.
- [X] I have [searched the current library](https://materialdesignicons.com/) to make sure the icon doesn't already exist.
- [X] I understand that [brand icons are no longer being added](http://dev.materialdesignicons.com/roadmap/brand-icons) and I can use [SimpleIcons](https://simpleicons.org/) for my brand icon needs.
- [X] I only intend to request a single icon (or a few near-identical ones) in this issue.

### What would you use this icon for?

Expressing RNG or randomness

I'm choosing a random page from a website, similar to Wikipedia's random article function. I like the dice you have which represent a dice that has already rolled, but an isometric die would represent the nondeterminism.

### Give us an example of what this icon might look like.

Basically a hexagon with a Y to make 3 visible faces.

I might implement the basic structure as an SVG myself but I don't trust my design to meet every standard you have.

**Comment by goyney** [comment]
Won't happen as it would violate [Material guidelines](https://material.io/design/iconography/system-icons.html#system-icon-metrics). Thanks for the suggestion.
